README.es-ES.md

```diff
@@ -66,7 +66,7 @@ Algunas de las herramientas en esta sección están aún en desarrollo y sujetas
 
 Bundling o empaquetado es una práctica estándar que permite reducir el número de solicitudes que el navegador necesita para entregar la aplicación solicitada por el usuario. En esencia, el "bundler" recibe una lista de puntos de entrada y los junta en uno o más bundles. De esta manera, el navegador puede obtener la aplicación completa realizando solo unas pocas solicitudes, en vez de ir solicitando de forma separada cada fichero.
 
-Así como crezca tu aplicación empaquetar todo en un único fichero puede ser contraproducente. Vea las técnicas de división de código usando Webpack.
+Así como crezca tu aplicación, empaquetar todo en un único fichero puede ser contraproducente. Vea las técnicas de división de código usando Webpack.
```

Comment:
En el texto en general y en este pa'rrafo en particular hay una mezcla de tuteo (_... como crezca **tu** aplicación..._) y el uso del [ustedeo](https://es.wikipedia.org/wiki/Ustedeo) (_Vea las técnicas ..._)

Reply:
Thank you!